packages/react-components/react-toast/src/state/types.ts

```diff
@@ -17,9 +17,16 @@ export interface ToastOptions {
   dispatchedAt: number;
 }
 
+export interface ToastOffsetObject {
+  horizontal?: number;
+  vertical?: number;
+}
```

Comment:
Using the same naming as in `usePositioning` does not make sense, but it's a bit strange to have different API 👀